components/Hero.js

```diff
@@ -20,4 +50,33 @@ const Hero = ({ children, gravity, src }) => {
 
 
 
+Hero.defaultProps = {
+  gravity: 'center',
+  size: 'half',
+  type: 'image',
+}
+
+Hero.propTypes = {
+  gravity: (props, propName, componentName) => {
```

Comment:
That's some next level prop validation right there.

Reply:
I was quite happy with it. 😉